KEYHOLDER.md

```diff
@@ -1,40 +1,84 @@
-# TUF Generation
+# Keyholder Responsibilities
 
-Pre-requisites:
-* A local Git installation and a Go development setup
-* SSH authentication for GitHub (see [here](https://docs.github.com/en/authentication/connecting-to-github-with-ssh))
-* Keyholders: A USB port connection for your hardware key (beware of using a remote connection; the keyholder should not assume that magic occurs during an SSH session)
+This document outlines the responsibilities of a root keyholder.
 
-0. **The keyholders and the conductor** should fork [this](https://github.com/sigstore/root-signing) git repository by clicking the "fork" button on GitHub. Then, set your `${GITHUB_USER}` with your GitHub username and execute the script:
+## Outline
 
-```
+Keyholders MUST subscribe to the [Sigstore Maintainer Calendar](https://calendar.google.com/calendar/u/0?cid=Y19ydjIxcDJuMzJsbmJoYW5uaXFwOXIzNTJtb0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t) for upcoming root signing event. Root signing events are expected to occur about every 4-5 months. The next `v+1` version signing will be scheduled, and the `v+2` version will be tentatively scheduled. Testing will occur the week before the signing. Keyholders are required to communicate that they have completed the [pre-work](#signing-pre-work) to the orchestrator through [#sigstore-keyholder](https://sigstore.slack.com/archives/C03E4HP6RCK) Slack channel. All testing and signing events can occur asynchronously. Keyholders are expected to be "on-call" during the root signing window in case there is an issue.
```

Comment:
Should we specify the hours and timezone that they should be available?

Reply:
Given that we're running during a two day window 1 month before expiration, I expect response sometime during that 2 day window and don't think a "to the hour" is necessary, given that people are across many, many timezones.